README.md

```diff
@@ -1,8 +1,8 @@
 # How to Get Started with the Syncfusion Vue Grid
 
-A quick start project that helps you to create and bind data to the Syncfusion Vue Grid. You will also learn how to customize tablular data and enable few of its important features like paging, sorting, filtering and grouping.
+The [Vue Grid](https://www.syncfusion.com/vue-components/vue-grid?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples), is a feature-rich control for displaying data in a tabular format. Its wide range of functionalities includes data binding, editing, Excel-like filtering, custom sorting, grouping, row reordering, freezing rows and columns, exporting to Excel, CSV and PDF formats, aggregating rows and more. A quick start project that helps you to create and bind data to the Syncfusion Vue Grid. You will also learn how to customize tablular data and enable few of its important features like paging, sorting, filtering and grouping.
 
-Documentation: https://ej2.syncfusion.com/vue/documentation/grid/getting-started/
+Documentation: https://ej2.syncfusion.com/vue/documentation/grid/getting-started
 
 Online example: https://ej2.syncfusion.com/vue/demos/#/material/grid/grid-overview.html 

```

Comment:
Remove doc and example link here since they had been added in related links

Reply:
Done